tests/Integration/Behaviour/Features/Scenario/CmsPage/cms_page_management.feature

```diff
@@ -0,0 +1,70 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s cms_page
+@reset-database-before-feature
+Feature: CmsPage Management
+  PrestaShop allows BO users to manage cms pages
+  As a BO user
+  I must be able to create, edit and delete cms page in my shop
+
+  Scenario: Adding new cms page
+    When I add new cms page "cmspage-1" with following properties:
+      | meta_title           | test.This is the title field          |
+      | head_seo_title       | headseotitle(legacy) or metatitle(new)|
+      | meta_description     |                                       |
+      | meta_keywords        | test,keyword,key2,key3                |
+      | link_rewrite         | friendlyurlexample                    |
+      | content              | <div> hello world </div>              |
```

Comment:
Even though it's a test, I'd like to see more meaningful data. For example not "test.This is the title field", but "My custom page" or similar, wdyt?